### PR TITLE
Data Release: 2022-06-29

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: internal/server/v0/triple/golden/get_triples/place_type.json [hunk 1]
```diff
@@ -3870,20 +3870,6 @@
       ],
       "provenanceId": "dc/5n63hr1"
     },
-    {
-      "subjectId": "wikidataId/Q27304761",
```
Comment (pradh) [inline internal/server/v0/triple/golden/get_triples/place_type.json:3874]: Expected - cl/446351803

**Review by pradh — APPROVED** [review]
Looks good!

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json [hunk 2]
```diff
@@ -1428,555 +1428,18 @@
         "Count_CriminalIncidents_WeaponLawViolations_IsHateCrime",
         "Count_CycloneEvent",
         "Count_CycloneEvent_TropicalStorm",
-        "Count_Death",
```
Comment (shifucun) [inline internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json:1431]: @pradh Can you confirm this deletion?

Comment (pradh) [reply internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json:1431]: Good eye.  Yes, that's from cl/453976812

**Pull request merged**